src/TournamentManager.ts

```diff
@@ -384,7 +386,7 @@ export class TournamentManager implements TournamentInterface {
 		const role = await this.discord.getPlayerRole(tournament);
 		let message = `A new round of ${tournament.name} has begun! ${this.discord.mentionRole(
 			role
-		)}\nPairings: ${url}`;
+		)}\nPairings will be send out by Direct Message shortly, or can be found here: ${url}`;
 		if (bye) {
 			message += `\n${this.discord.mentionUser(bye)} has the bye for this round.`;
 		}
```

Comment:
I assume you're referring to this but this is an automatic bye that results from having an odd number of players rather than the manual round-one byes. I agree that this is probably optional but we can solicit feedback on this.

Reply:
It seems you have solicited feedback in the meantime, and the client would prefer for this to be shifted to DMs, so I'll make that change.